src/main/java/core/basesyntax/model/machine/Machine.java

```diff
@@ -1,9 +1,31 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.Table;
+
+@Entity(name = "machines")
+@Table(name = "machines")
```

Comment:
are you sure you need two names

Reply:
not sure

Reply:
I am also not sure. Seems like Enity name = alias for class for Hibernate. But table name is name of table.